packages/neo-one-cli/src/interactive/createCRUD.js

```diff
@@ -151,22 +153,24 @@ const renderTasks = (
 };
 
 const promptDelete = ({
-  cli,
   crud,
   name,
 }: {|
-  cli: InteractiveCLI,
   crud: CRUDResource<*, *>,
   name: string,
-|}) =>
-  cli.vorpal.activeCommand.prompt({
-    type: 'confirm',
-    name: 'continue',
-    default: false,
-    message: `Are you sure you want to delete ${
-      crud.resourceType.name
-    } ${name}?`,
-  });
+|}) => {
```

Comment:
nit: I'd leave it the way it was before, just replace `cli.vorpal.activeCommand.prompt` with `inquirer.prompt`